### PR TITLE
Improve the homepage title

### PR DESCRIPTION
**Pull request by peteryates**
This is a test PR

### CODE REVIEW

File: app/views/content/home.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
   layout: "layouts/home"
   title: "Inspire the next generation"
   description: |-
-    Read official Department for Education information to help you become a teacher. Learn about funding, teacher training, and steps to become a teacher. 
+    Become a teacher today!
```
Comment (KeishaDfE) [inline app/views/content/home.md:5]: Great line

File: app/views/content/home.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
   layout: "layouts/home"
   title: "Inspire the next generation"
   description: |-
-    Read official Department for Education information to help you become a teacher. Learn about funding, teacher training, and steps to become a teacher.
```
Comment (GDW7) [inline app/views/content/home.md:5]: Don't do it :)

Comment (jademariefleuriot) [inline app/views/content/home.md:5]: This works

File: app/views/content/home.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
   layout: "layouts/home"
   title: "Inspire the next generation"
```
Comment (KeishaDfE) [inline app/views/content/home.md:3]: good

**Review by jademariefleuriot — COMMENTED** [review]
Happy with the review

**Review by KeishaDfE — COMMENTED** [review]
Homepage copy to be updated

File: app/views/content/home.md (hunk 1 above)
Comment (jademariefleuriot) [inline app/views/content/home.md:5]:
```suggestion
    Become a teacher now! 
```

**Comment by peteryates** [comment]
Closing 💥

**Pull request closed without merging**